ec2system/ec2machine_test.go

```diff
@@ -76,15 +76,15 @@ func TestMutualHTTPS(t *testing.T) {
 
 	var listenAndServeError errors.Once
 	go func() {
-		listenAndServeError.Set(sys.ListenAndServe(fmt.Sprintf(":%d", port), mux))
+		listenAndServeError.Set(sys.ListenAndServe(fmt.Sprintf("127.0.0.1:%d", port), mux))
 	}()
 	time.Sleep(time.Second)
 
 	config, _, err := authority.HTTPSConfig()
 	transport := &http.Transport{TLSClientConfig: config}
 	http2.ConfigureTransport(transport)
 	client := &http.Client{Transport: transport}
-	_, err = client.Get(fmt.Sprintf("https://localhost:%d/", port))
+	_, err = client.Get(fmt.Sprintf("https://127.0.0.1:%d/", port))
```

Comment:
Is there a reason to prefer 127.0.0.1 throughout instead of localhost? I'm not sure if this has any IPv4 vs. IPv6 implications.

Reply:
I guess in theory someone can have localhost defined as something other than 127.0.0.1, but then you probably have bigger problems.

Reply:
`localhost` it is.